admin-dev/themes/default/js/bundle/module/module.js

```diff
@@ -44,6 +44,12 @@ var AdminModuleController = function() {
   this.categoryGridItemSelector = '.module-category-item';
   this.addonItemGridSelector = '.module-addons-item-grid';
   this.addonItemListSelector = '.module-addons-item-list';
+  
+  // Upgrade All selectors
+  this.upgradeAllSource = '.module_action_menu_upgrade_all';
+  this.upgradeAllTargets = '#modules-list-container-update .module_action_menu_upgrade';
```

Comment:
I've just discovered that if we spam the "Update all" button, the request is sent multiple time.
When a request is sent, the buttons are hidden. We can use their visibility to check if the previous one is finished.